content/articles/building-a-video-cataloging-app-with-react-firebase-and-styled-components/index.md

```diff
@@ -0,0 +1,543 @@
+
+### Guide to React development cycle
+The first thing to have in mind when building web applications with React.js is creating the React application. This process is done by running a single command in the `command terminal` of your computer. The command scaffolds the application creation by setting up the required files, folders, and default dependencies needed to start development.
+
+#### Step 1 - Creating the React application
+To create our React application, we open up the command terminal and run the command as shown below:
+
+```bash
+npx create-react-app My-app
+```
+
+Alternatively, for yarn users:
+
+```bash
+yarn create-react-app My-app
+```
+
+The process requires some minutes and an internet connection to set up the new application so ensure your computer is connected to an active Wi-Fi or Modem.
+Once the application has been created, it’s time to begin development.
+
+#### Step 2 - Installing the required dependencies
+We need to install some dependencies to enable our application to perform the desired tasks. And they include the following:
+
+- Firebase: the firebase package will enable our application to have access to the Google cloud database suite where our content will be stored and managed. Firebase has numerous functionalities including authentication, hosting, and other server management features. Feel free to look up the [official documentation](https://firebase.google.com/) for more information.
+- [React-styled-components](https://www.styled-components.com): this will be used to style and customize our components. React-styled-components reduces the complexities of styling React applications since it allows you to target specific page elements for styling. It also allows the flexibility of writing both the JavaScript and the styles snippets in one `.js` file.
+  To install the above dependencies to our application, open the `package.json` file, and add the snippet shown below:
+
+```JSON
+"dependencies": {
+   "firebase": "^8.4.1",
+   "styled-components": "^5.2.3"
+},
+```
+
+Thereafter we install them by running the command below:
+
+```bash
+npm install
+```
+
+Alternatively, for yarn:
+
+```bash
+yarn add
+```
+
+That should start the installation process. Once that is completed, it is time to begin the application setup.
+
+#### Step 3 - Setting up Firebase (firebase.js)
+To set up and use the firebase suite in our application, there are some basic steps we need to fulfill to get the config keys for the application. To get started, go to the [Google Firebase website](https://firebase.google.com/) and create a new project. Feel free to give the new project a name of your choice. Once that is completed, the required config keys will be revealed in the console and you should copy them. Thereafter, in the `src` folder of your React app, create a new `firebase.js` file, thereafter you go ahead and paste the config copied in the `firebase.js` file. it should look similar to this below:

```

Comment:
The instructions you give here regarding obtaining the required config keys are misleading and incorrect. 

In the case that you don't wish to go into details about a certain topic, consider sending your readers to [official documentation](https://firebase.google.com/docs/web/setup) that will guide them on what to do.

Reply:
i updated the instructions, it didnt reflect because the PR is closed